storagemanager/storage/static/css/main.css

```diff
@@ -93,11 +93,16 @@ button:active {
 }    
 
 .all_items {
-    width: 750px;
+    width: auto;
     margin: auto;
-    text-align: center;
+    text-align: left;
+    border: 1px white solid;
+    padding-left: 38%;
 
 }
+
+
+
 small.id {
     size: 4pt;
```

Comment:
Is there any reason to use points here? Better not mix pixels and points, let's stick to pixels only in css.